modelforge/potential/utils.py

```diff
@@ -5,6 +5,48 @@
 import torch.nn as nn
 
 
+class SlicedEmbedding(nn.Module):
+    def __init__(
```

Comment:
Can you add in a docstring for this class?

Reply:
sure!